src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -159,7 +160,13 @@ private void OnLoaded(object sender, RoutedEventArgs e)
 
             SearchBox.QueryTextBox.DataContext = _viewModel;
             SearchBox.QueryTextBox.PreviewKeyDown += Launcher_KeyDown;
-            SearchBox.QueryTextBox.TextChanged += QueryTextBox_TextChanged;
+
+            Observable.FromEventPattern<TextChangedEventHandler, TextChangedEventArgs>(
+                add => SearchBox.QueryTextBox.TextChanged += add,
+                remove => SearchBox.QueryTextBox.TextChanged -= remove)
```

Comment:
What does this do? Makes no sense for me to add something and the remove it.

Reply:
Hey @htcfreek. This is just standard semantics for creating an observable from an event in Rx.